sphinxcontrib/video.py

```diff
@@ -150,6 +152,8 @@ def run(self) -> List[video_node]:
                 preload=preload,
                 width=width,
                 klass=self.options.get("class", ""),
+                align=self.options.get("align", ""),
```

Comment:
If you are going to use "default" everywhere else why not using it as default value ? 

```suggestion
                align=self.options.get("align", "default"),
```